tools/ci/setup_docker_dataverse.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+# This is setting up the (already running!) server with everything needed for
+# our tests to run against this instance.
+# Most importantly with respect to the CI builds, it deposits the API tokens
+# for the users `pete` and `uma` in environment variabes TEST_TOKEN_PETE and
+# TEST_TOKEN_UMA respectively for use from within unit tests. Therefore this
+# script is supposed to be sourced.
+
+# For administrative tasks at the beginning, we need to execute request from
+# within the container, since the respective API endpoints are only accessible
+# from 'localhost' (from the POV of the dataverse server). Hence, finding the
+# container ID and running via `docker exec` is required.
+
+# Note, that there are a couple more setup scripts already available in the
+# container in /opt/payara/dvinstall besides the currently used `setup-users.sh`.
+# We may want to use more of them.
+# Consider especially init-setup.sh, setup-dvs.sh
+
+### Initial container setup
+git clone https://github.com/IQSS/dataverse-docker
+docker network create traefik
+cp dataverse-docker/.env_sample dataverse-docker/.env
+docker-compose --file dataverse-docker/docker-compose.yml --env-file dataverse-docker/.env up -d
+# On AppVeyor the server takes a while to become responsive.
+# Note: We probably want to integrate the containers' `init-setup.sh`, which
+# implements a way to keep probing the server until it's up rather than
+# waiting a fix amount of time.
+sleep 300
```

Comment:
Agree. However, not sure that will change anything for the CI. I had AppVeyor failing consistently because of that and saw it taking more than 4 minutes.